examples/src/main/java/io/dapr/examples/secrets/README.md

```diff
@@ -61,18 +61,23 @@ Dapr's API for secret store only support read operations. For this sample to run
 vault login myroot
 ```
 
-> Note: If you get `http: server gave HTTP response to HTTPS client` make sure the local vault address is set `export VAULT_ADDR=http://127.0.0.1:8200/`
+> Note: If you get `http: server gave HTTP response to HTTPS client` make sure to set the local vault address as  `export VAULT_ADDR=http://127.0.0.1:8200/`
 
 2. Create secret (replace `[my favorite movie]` with a title of our choice):
 ```bash
 vault kv put secret/dapr/movie title="[my favorite movie]"
 ```
 
+3. Create random secret:
+```bash
+vault kv put secret/dapr/randomKey testVal="value"
```

Comment:
May be use a name like `disallowedKey`? and `testVal="to demonstrate exclusion from secret allow list"` to be more descriptive?

Reply:
A specific key is not disallowed here ... rather only a specific key is allowed.